### PR TITLE
Cloud.gov deployments setup

### PR DESCRIPTION
**Pull request by loganmcdonald-noaa**
# What

This sets up `beta`, `staging`, and `test` in order to deploy to them. It also sets up a series of documentation and scripts for deploying and running post-deploy steps.

### CODE REVIEW

File: docs/dev/deploying.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Deploying
+
+We usually deploy using the Github Actions provided. You can see more about how deploys with in each environment [here](../environments.md).
```
Comment (echappen) [inline docs/dev/deploying.md:3]: [non-blocking question] Typo?

File: manifests/manifest-beta.yaml [hunk 2]
```diff
@@ -17,11 +17,11 @@ applications:
...
-  no-route: true
-  command: ./cronish.sh
-  health-check-type: process
-  memory: 128M
+# - name: cronish
+#   <<: *defaults
+#   no-route: true
+#   command: ./cronish.sh
+#   health-check-type: process
+#   memory: 128M
```
Comment (echappen) [inline manifests/manifest-beta.yaml:27]: [non-blocking question] can this commented-out code be removed? (same question for all manifest files)

File: scripts/create-cloudgov-env.sh [hunk 3]
```diff
@@ -0,0 +1,139 @@
+# This script sets up a completely new Cloud.gov CF Space with all the corresponding
+# infrastructure needed to run weahter.gov. It can serve for documentation for running
```
Comment (echappen) [inline scripts/create-cloudgov-env.sh:2]: [non-blocking comment] typo weather

File: scripts/create-cloudgov-env.sh [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+generate_string "$CRON_KEY"
+CRON_KEY=${CRON_KEY:-$NEW_STRING}
+generate_string "$HASH_SALT"
+HASH_SALT=${HASH_SALT:-$NEW_STRING}
+generate_string "$ROOT_USER_PASS"
+ROOT_USER_PASS=${ROOT_USER_PASS:-$NEW_STRING}
+ROOT_USER_NAME=${ROOT_USER_NAME:-root}
+cf create-user-provided-service secrets -p "{\"CRON_KEY\": \"$CRON_KEY\", \"HASH_SALT\": \"$HASH_SALT\", \"ROOT_USER_NAME\": \"$ROOT_USER_NAME\", \"ROOT_USER_PASS\": \"$ROOT_USER_PASS\"}"
+
+echo "Database create succeeded and credentials created. Deploying the get.gov application to the new space $1..."
```
Comment (echappen) [inline scripts/create-cloudgov-env.sh:87]: [non-blocking question] weather.gov instead of get.gov?

**Review by echappen — APPROVED** [review]
No comments on functionality, just a few wording nits, all non-blocking. Otherwise LGTM 👍

File: manifests/manifest-beta.yaml (hunk 2 above)
Comment (loganmcdonald-noaa) [reply manifests/manifest-beta.yaml:27]: It's something we definitely want in the future but it's not something we are using currently.

File: scripts/create-cloudgov-env.sh (hunk 4 above)
Comment (loganmcdonald-noaa) [reply scripts/create-cloudgov-env.sh:87]: Yep :) still stuck in previous project headspace I guess!

**Pull request merged**